src/raw.imageio/rawinput.cpp

```diff
@@ -1505,7 +1505,7 @@ RawInput::read_native_scanline(int subimage, int miplevel, int y, int /*z*/,
         // to the array direction so we must copy the pixels into a temporary contiguous buffer
         else if (sizes.flip == 5 /*90 degrees CCW*/
                  || sizes.flip == 6 /*90 degrees CW*/) {
-            scanline_start = m_spec.height - y + sizes.left_margin;
+            scanline_start = m_spec.height - 1 - y + sizes.left_margin;
```

Comment:
Does the `scanline_start` expression on Line 1496 also need the `-1` adjustment?

Reply:
I believe you are correct, I will fix there as well